bears/verilog/VerilogLintBear.py

```diff
@@ -24,6 +24,7 @@ class VerilogLintBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Redundancy', 'Simplification', 'Syntax'}
```

Comment:
``It also warns about unoptimized code due to some construct, with which the
     optimization of the specified signal or block is disabled.`` is this not simplification? @sils1297